gc/gc-repository-jdbc/src/testFixtures/java/org/projectnessie/gc/contents/jdbc/AbstractJdbcPersistenceSpi.java

```diff
@@ -58,18 +57,18 @@ protected PersistenceSpi createPersistenceSpi() {
   @BeforeEach
   void setup() throws Exception {
     try (Connection conn = dataSource.getConnection()) {
-      JdbcHelper.createTables(conn);
+      JdbcHelper.createTables(conn, false);
+      // Test table creation with existing tables
```

Comment:
Should this actually be a test? Not sure whether we want to call `createTables` two times before each test case